packages/loader/container-utils/src/error.ts

```diff
@@ -55,17 +54,43 @@ function extractLogSafeErrorProperties(error: any) {
 }
 
 /**
- * Generic error
+ * Generic wrapper for an unrecognized/uncategorized error object
  */
 export class GenericError extends LoggingError implements IGenericError {
     readonly errorType = ContainerErrorType.genericError;
 
     constructor(
-        errorMessage: string,
-        readonly error: any,
+        message: string,
         props?: ITelemetryProperties,
```

Comment:
this change is risky, as i don't think you'll get a compile time error for any missed cases

Reply:
Hmmmmm good point, I do remember getting test failures for it.  I'll double-check usages. I don't imagine anyone outside our codebase is using this constructor either...

Reply:
Reverted it, thanks for pointing this out.